developers/weaviate/manage-data/read-all-objects.mdx

```diff
@@ -75,7 +75,7 @@ The following code iterates through all objects, providing the properties and id
   </TabItem>
 </Tabs>
 
-## Read all objects with Vectors
+## Read all objects with vectors
```

Comment:
Consider 
Read all objects that have vectors

This one feels ok, but the one on ln 112 sounds like it uses tenants to read objects. Updating this one too for parallel structure.

Reply:
Updated. FYI the distinction isn't necessary whether the objects have vectors, but whether to retrieve the vectors or not.